dask/dataframe/core.py

```diff
@@ -1543,7 +1543,12 @@ def describe(self, split_every=False, percentiles=None):
         if percentiles is None:
             percentiles = [0.25, 0.5, 0.75]
         else:
-            percentiles = sorted(set(list(percentiles) + [0.5]))
+            # always include the the 50%tle to calculate the median
+            # unique removes duplicates and sorts quantiles
+            percentiles = np.array(percentiles)
+            percentiles = np.append(percentiles, 0.5)
```

Comment:
why do we always need the median?

Reply:
I don't know. It's just how pandas does it. It is unexpected though so I thought a comment clarifying why the 0.5 comes out of nowhere couldn't hurt.

Reply:
Oh indeed, you are right - that's fine then, I suppose.